stdlib/nashorn/file.rb

```diff
@@ -0,0 +1,13 @@
+class File < IO
```

Comment:
Can you remove the `< IO` part just to avoid conflicts if the superclass of `File` ever changes?